posthog/middleware.py

```diff
@@ -223,6 +223,11 @@ def __init__(self, get_response):
 
     def __call__(self, request: HttpRequest):
         if request.path == "/decide/" or request.path == "/decide":
```

Comment:
Q: Why do we short circuit decide but not capture? cc @xvello you have been working with these middlewares a lot - is this the right place to be even tagging things?

Reply:
Heya. I have [a PR open](https://github.com/PostHog/posthog/pull/13319) to also shortcut on capture, but I suspended it due to (relevant) concerns about losing the observability.

Today showed us that we do lack observability on what is keeping the gunicorn workers busy.